### DISCUSSION TITLE
Conditional rendering of headers for multiple listings

### DISCUSSION THREAD

**Discussion by l-gorman** [quarto-dev/quarto-cli#10433] [Q&A]
### Description

Been using quarto a lot recently, so thank you for everything!

I'm creating a page that uses multiple listings, as shown in documentation [here](https://quarto.org/docs/websites/website-listings.html#multiple-listings).

It looks like this:

![image](https://github.com/user-attachments/assets/727b76fc-1293-40ef-b30a-744bac8ee8ab)

When I use some filters, some listings do not contain any records covered by the selection of filters (as expected), see here:

![image](https://github.com/user-attachments/assets/7d81216d-a107-42f5-9725-690d35ac05ae)

It would be really helpful if the headers/listings could be rendered only in cases where they contain matching cards. Any thoughts of how to do this, or if there is a way to do this would be really helpful.

You can find my code here:
https://github.com/uob-training/uob-training.github.io

**Comment by mcanouil** [comment]
The header is unrelated to the listing.

This is the listing:

```
:::{#fundamentals}
:::
```

This is what you hardcoded independently of the listing:

```
## Fundamentals

These are courses which are not specific to
any programming language. They are useful skills 
to have regardless of the language you are using.
```

Quarto cannot know the two are tied together in your setup.

I suggest you define your own custom listing in which you add the headers which in there could be conditioned of the presence of at least one item.
See <https://quarto.org/docs/websites/website-listings-custom.html>.
